Scripts/Playthrough.cs

```diff
@@ -182,14 +197,18 @@ public async void Connect(Action onReadyCallback)
         // Disconnect from the current interaction.
```

Comment:
Sometimes the server will disconnect without the client asking it to (i.e. without the client calling `Disconnect`), in order to listen for this we should subscribe to `room.OnLeave()` (just below the `room.OnMessage` listeners above), and if this fires, it should `SetConnectionState(ConnectionState.Disconnected)` too